database/create_backup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Make sure to define IS_DEV_ENV before running this script
+if [[ -z "${IS_DEV_ENV}" ]]; then
+    echo "IS_DEV_ENV undefined, please set IS_DEV_ENV environment variable." 1>&2
+    exit 1
```

Comment:
Forslag til endring:

- Scriptet burde selv lette etter variabelen i `.env` filen.
- Dersom IS_DEV_ENV ikke er definert i `.env` filen, eller at `.env` filen ikke fins ,  så kan scriptet anta at `IS_DEV_ENV = true`

Reply:
Thanks for review, this is a good idea! I will implement this

Reply:
Alright jeg har implementert dette nå, håper det funker bra :D